lib/datasource/pypi/index.ts

```diff
@@ -83,17 +83,33 @@ async function getDependency(
       }
     }
 
-    if (dep.info?.project_urls && !dependency.sourceUrl) {
-      for (const [name, projectUrl] of Object.entries(dep.info.project_urls)) {
-        const lower = name.toLowerCase();
-        if (
-          lower.startsWith('repo') ||
-          lower === 'code' ||
-          lower === 'source' ||
-          github_repo_pattern.exec(projectUrl)
-        ) {
-          dependency.sourceUrl = projectUrl;
-          break;
+    if (dep.info?.project_urls) {
+      if (!dependency.sourceUrl) {
```

Comment:
shouldn't the if combined to the inner condition?

Reply:
sorry, wrong if condition